packages/editor-ui/src/constants.ts

```diff
@@ -2,7 +2,10 @@ import type { NodeCreatorOpenSource } from './Interface';
 import { NodeConnectionType } from 'n8n-workflow';
 
 export const MAX_WORKFLOW_SIZE = 16777216; // Workflow size limit in bytes
-export const MAX_WORKFLOW_PINNED_DATA_SIZE = 12582912; // Workflow pinned data size limit in bytes
+export const MAX_EXPECTED_REQUEST_SIZE = 2048; // Expected maximum workflow request metadata (i.e. headers) size in bytes
```

Comment:
```suggestion
export const MAX_REQUEST_METADATA_SIZE = 2048; // Expected maximum workflow request metadata (i.e. headers) size in bytes
```

Reply:
This is the actual **expected** maximum size that we take into account and it's not enforced. It's only used for calculating if we exceed the maximum request size. It's according to some statistics from Google.